packages/app/src/PasswordCheck.tsx

```diff
@@ -0,0 +1,70 @@
+import { useMemo } from 'react';
+
+const checkLength = (password: string) => password.length >= 12;
+const checkOneUpper = (password: string) => /[A-Z]+/.test(password);
+const checkOneLower = (password: string) => /[a-z]+/.test(password);
+const checkOneNumber = (password: string) => /\d+/.test(password);
+const checkOneSpecial = (password: string) => /\W+/.test(password);
+
+export const PasswordCheck = (password: string | null) => {
+  password = password ?? '';
+  return (
+    <ul>
+      <li>
```

Comment:
personally not a fan of the li's here, can probably do without the bullet points and just have them as just divs to make it nice and clean

<img width="497" alt="image" src="https://github.com/hyperdxio/hyperdx/assets/2781687/5b4e4c05-556b-49c5-bc67-9110ab0ba5fc">

Reply:
Done and done, will roll it into the `watch` commit